sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -118,6 +118,15 @@ int32_t RetryPolicy::GetRetryCount(Context const& context)
   return *ptr;
 }
 
+bool RetryPolicy::ShouldRetry(
+    std::unique_ptr<RawResponse> const& response,
+    RetryOptions const& retryOptions) const
+{
+  (void)response;
```

Comment:
Are you planning on documenting this function? If that's the case, keeping these named parameters makes sense, but if you're not planning on documenting this function, I suggest removing the named parameters, which avoids the ugly cast to `void`.

Reply:
Agreed. It's already documented in the public header file. I won't be documenting it here in the source, so will update accordingly.